src/pages/about.js

```diff
@@ -1,7 +1,6 @@
 // Step 1: Import your component
 import * as React from 'react'
 import Layout from '../components/layout'
-import {link} from 'gatsby'
```

Comment:
Make sure you clean up your unused code when you're done working. If you're using VSCode as your editor, or use some external code linter, it should tell you any unused variables / imports and where to find them. (Example in screenshot)

<img width="879" alt="unused-variable" src="https://user-images.githubusercontent.com/50115852/127189312-e509588a-c9f2-45b1-92be-d5e822c1ade8.png">

Reply:
navContainer is not unsed in lauout.js file.
![image](https://user-images.githubusercontent.com/22979867/127212886-5d87d37a-33e4-47be-92ef-c46f3d7669c3.png)

Reply:
Oh yes, I just quickly deleted something so you could see how the warning would show up in the console. It was just an example 👍

Reply:
Ah gotcha!